Libbna/CUSTOM-CMS
# #214: fixed message showing on block and menu add.

**Testing Steps:**

- Add any new menu and see message appear on top
- Add any new block and see message appear on top
- Refer Below Screenshots

![image](https://user-images.githubusercontent.com/32816725/149318323-530631d3-74c4-417b-ac9d-b6206c18989b.png)
